rs/replica/setup_ic_network/src/lib.rs

```diff
@@ -362,12 +369,14 @@ fn start_consensus(
         );
 
         join_handles.push(jh);
-        new_p2p_consensus.add_client(
-            ingress_rx,
+        let downloader = ic_artifact_downloader::FetchArtifact::new(
```

Comment:
Shouldn't these be renamed to assembler?

Reply:
Yes. Missed them.